Courses/Views/Components/CourseItem.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  CourseItem.swift
+//  Courses
+//
+//  Created by Dmytro Churakov on 20.10.2022.
+//
+
+import SwiftUI
+
+struct CourseItem: View {
+    var namespace: Namespace.ID
```

Comment:
@DCHRKV don't forget to set correct access

Reply:
@Den-Ree , I can't change it to private because I need to initialize it in preview